### ISSUE TITLE
Models not training

### ISSUE THREAD

**Issue by usmananjum** [cnguyen10/few_shot_meta_learning#23]
Hi, 

I like your repository and the code you have implemented. I am facing a couple of issues:

The dataset - i cannot find the dataset in the format you have asked. I have tried using the dataloader from torchmeta to run with your code but the issue is that most of your code when run does not go beyond 20.00 accuracy. Do you have any advice on what I may do?

**Comment by cnguyen10** [comment]
If it showed 20% for 5-way classification setting, it did mean that the model was not learnt. Given your vague description, it is difficult to know where things are wrong. Could you provide a code snippet to explain what you did?

**Comment by cnguyen10** [comment]
By the way, did you use your own model or the one provided in 'CommonModels.py'? I just recall that I faced a similar issue in the past where my model did not learn at all. The issue was due to batchnorm. Without disabling all the batchnorm layers, the model got stuck at 1.61 loss and 20% accuracy.

**Comment by usmananjum** [comment]
Thank you for the response. 

Can you provide a link to the datasets? Maybe it might be an issue with the dataloader I created.

**Comment by cnguyen10** [comment]
I already specified the structure of the dataset in the `readme.md`. The dataset used in this case simply contains many folders, each consists of several images from the same class.
